test/performance/range/views/view_minimiser_hash_benchmark.cpp

```diff
@@ -130,6 +132,37 @@ void compute_minimisers(benchmark::State & state)
     state.counters["Throughput[bp/s]"] = bp_per_second(sequence_length - k + 1);
 }
 
+template <method_tag tag>
+void compute_minimisers_on_poly_A_sequence(benchmark::State & state)
+{
+    auto sequence_length = state.range(0);
+    size_t k = static_cast<size_t>(state.range(1));
+    uint32_t w = static_cast<size_t>(state.range(2));
+    assert(sequence_length > 0);
+    assert(k > 0);
+    assert(w > k);
+    auto seq = seqan3::views::repeat_n(seqan3::dna4{}, sequence_length) | seqan3::views::to<std::vector>;
```

Comment:
```suggestion
    auto seq = std::vector<seqan3::dna4>(sequence_length);
```

I had a presentation about this :(

https://en.cppreference.com/w/cpp/container/vector/vector